dotcom-rendering/src/model/article-schema.json

```diff
@@ -321,6 +321,24 @@
         "hasStoryPackage": {
             "type": "boolean"
         },
+        "storyPackage": {
```

Comment:
Frontend now sends this data to DCR (thanks to https://github.com/guardian/frontend/pull/25267)